release_notes/45-release-notes.adoc

```diff
@@ -404,4 +405,27 @@ This release includes notable technical changes and bug fixes.
 * After upgrading to {product-title-short} 4.5.0, for nodes with more than 64 cores, Collector displayed an error about the ring buffer size not being allowed. This problem has been fixed.
 * The network graph was broken for users with clusters running Google Kubernetes Engine (GKE) version 1.29 and later due to a change from Google to use the managed public IP address range `34.118.224.0/20` for default networking services. The network graph marked this IP address range as external. This problem has been fixed.
 
-include::modules/image-versions.adoc[leveloffset=+1]
+[id="about-release-452_{context}"]
+== About release 4.5.2
+
+*Release date*: 11 September 2024
+
+This release of {product-title-short} introduces the following change:
+
+* You can modify the CVE management process to temporarily move orphaned CVEs to a tombstone table to prevent immediate removal and preserve accurate discovery times.
+
+This release of {product-title-short} fixes the following bugs:
+
+* Fixed an issue where Sensor was unable to identify {ocp} internal registry secrets due to a change in the pull secret annotation.
+
+* Fixed an issue where policies incorrectly displayed an enforced value of `Yes` in the *Configuration Management* -> *Application & Infrastructure* -> *Deployments* page due to inconsistent status handling, while showing `No` in the *Violations* page.
+
+* Fixed an issue where vulnerability data differed between the Vulnerability Management Workload CVE pages and the deprecated Vulnerability Management dashboard due to inconsistencies in vulnerability reporting.
+
+* Fixed an issue with database migration where namespace exclusions were added to default policies, causing the removal of the category associated with the root user policy.
```

Comment:
@kcarmichael08 we are not shipping database migration in 4.5.2. we will ship it in 4.6